scripts/run_tests.sh

```diff
@@ -9,17 +9,13 @@ set -e
 flake8 --max-complexity 15 /app
 mypy --ignore-missing-imports /app
 bandit -r /app
-rm -rf /spec
```

Comment:
I wasn't able to find the purpose of these lines if the tests use `sample_spec_release`

Small thing, but we can do `mkdir -p /spec/repo`